ovh/data_dbaas_logs_test.go

```diff
@@ -0,0 +1,47 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const testAccDataSourceDbaasLogs_dedicated = `
+data "ovh_dbaas_logs" "ldp" {
```

Comment:
When we test datasource it is recommended to first create a resource, then the datasource.
Because if the user running the test don't have a ldp, the test will failed.
And what about when the acceptance tests are ran in our CI/CD tool? :) 

Example to follow:
https://github.com/ovh/terraform-provider-ovh/blob/master/ovh/data_cloud_project_database_test.go

Thanks :)

Reply:
AFAIK, there is no api to create a LDP resource. It can only reference a manually created, existing one.

Reply:
so you can't test on computed resource, only the existence of the datasource, and on its service name for example :)